packages/react-core/src/components/DataList/DataListToggle.tsx

```diff
@@ -40,7 +40,9 @@ export const DataListToggle: React.FunctionComponent<DataListToggleProps> = ({
         aria-labelledby={ariaLabel !== 'Details' ? null : `${rowid} ${id}`}
         aria-expanded={isExpanded}
       >
-        <AngleRightIcon />
+        <div className="pf-c-data-list__toggle-icon">
```

Comment:
Why div, and why is the `pf-c-data-list__toggle-icon` hard coded?

Reply:
Replaced hard-coded className with react-styles variable.  This was the only change called out to use a `div` rather than `span` in the origin issue:

> The icon in the button in .pf-c-data-list__toggle should be wrapped with a new element with class .pf-c-data-list__toggle-icon - wrapper should be a <div>